Tasks/AzureRmWebAppDeployment/package.json

```diff
@@ -22,6 +22,8 @@
     "vsts-task-lib": "^0.9.7",
     "q": "^1.4.1",
     "adal-node": "^0.1.15",
-    "xml2js": "^0.4.13"
+    "xml2js": "^0.4.13",
+    "archiver":"^1.1.0",
+    "randomstring":"^1.1.5"
```

Comment:
check OSS approvals for the newly added modules and get thirdpartylicenses updated.